Resources/config/services.yml

```diff
@@ -1,4 +1,33 @@
+# This file is part of the CampaignChain package.
+#
+# (c) CampaignChain Inc. <info@campaignchain.com>
+#
+# For the full copyright and license information, please view the LICENSE
+# file that was distributed with this source code.
+
+parameters:
+    campaignchain.activity.gotowebinar.add:
+        bundle_name: "campaignchain/activity-gotowebinar"
+        module_identifier: "campaignchain-gotowebinar"
+        location: %campaignchain.location.slideshare.user%
+        equals_operation: true
+        operations:
+            - %campaignchain.operation.slideshare.schedule%
+        handler: "campaignchain.activity.controller.handler.gotowebinar.add"
+
 services:
-#    campaign_chain_activity_go_to_webinar.example:
-#        class: CampaignChain\Activity\GoToWebinarBundle\Example
-#        arguments: [@service_id, "plain_value", %parameter%]
+    campaignchain.activity.controller.slideshare.schedule:
+        class: CampaignChain\CoreBundle\Controller\Module\ActivityModuleController
+        calls:
+            - [setContainer, ["@service_container"]]
+            - [setParameters, ["%campaignchain.activity.slideshare.schedule%"]]
```

Comment:
@groganz is this right here, the parameter you pass in? cause it's slideshare.schedule and has nothing to do with webinar